packages/emitter-framework/test/typescript/components/interface-declaration.test.tsx

```diff
@@ -564,7 +564,7 @@ describe("Typescript Interface", () => {
         const actualContent = await format(testFile.contents as string, { parser: "typescript" });
         const expectedContent = await format(
```

Comment:
I want to remove the format calls and redo the entire test file to use the new pattern, but will split that off separately